locales/it/index.yml

```diff
@@ -1435,6 +1435,10 @@ wallet:
       inserisci manualmente i dati (Codice Avviso e Codice Fiscale Ente Creditore).
     setManually: Inserisci manualmente
     enroll_cta: Per inquadrare il codice QR presente sull'avviso, devi prima consentire ad IO l'uso della fotocamera. Puoi farlo dal menu 'Impostazioni' del tuo dispositivo.
+    readStorageDisclosure:
+      title: Consenti l'accesso alla galleria
+      message: Potrai caricare screenshot e avvisi di pagamento
+      cta: Scegli
```

Comment:
Since this key is not used, can we remove it?

Reply:
Sure I totally forgot to remove it
https://github.com/pagopa/io-app/pull/3563/commits/f77c5dc4ef4e16f6bb1f2947db2db41665db28a9